src/worker.js

```diff
@@ -13,7 +13,10 @@ async function worker(options) {
     filename: options.filename,
     warnings: [],
     errors: [],
-    info: {},
+    info: {
+      sourceFilename:
+        (options.info && options.info.sourceFilename) || options.filename,
```

Comment:
Better to check on `typeof sourceFilename === "string"` (same for filename), because `sourceFilename` is string, it can be an empty string, it doesn't make sense in our case, but it's better to do it right

Reply:
`typeof` checks have been added